lib/pacemaker/pcmk_sched_bundle.c

```diff
@@ -555,18 +555,28 @@ pcmk__bundle_with_colocations(const pe_resource_t *rsc,
     }
 }
 
+/*!
+ * \internal
+ * \brief Return action flags for a given bundle resource action
+ *
+ * \param[in,out] action  Bundle resource action to get flags for
+ * \param[in]     node    If not NULL, limit effects to this node
+ *
+ * \return Flags appropriate to \p action on \p node
+ */
 enum pe_action_flags
 pcmk__bundle_action_flags(pe_action_t *action, const pe_node_t *node)
 {
     GList *containers = NULL;
     enum pe_action_flags flags = 0;
```

Comment:
We should use `uint32_t` here and in `pcmk__collective_action_flags()`. Can be multiple flags.

Reply:
I'm pretty sure I meant to do that in a separate commit and forgot